cinema/tests/test_movie_api.py

```diff
@@ -147,9 +141,7 @@ def test_filter_movies_by_title(self):
     def test_retrieve_movie_detail(self):
         movie = sample_movie()
         movie.genres.add(Genre.objects.create(name="Genre"))
-        movie.actors.add(
-            Actor.objects.create(first_name="Actor", last_name="Last")
-        )
+        movie.actors.add(Actor.objects.create(first_name="Actor", last_name="Last"))
```

Comment:
Ensure that the `Actor` and `Genre` objects are correctly associated with the `Movie` object in the database. This is important for the `test_retrieve_movie_detail` test to pass successfully.